Makefile

```diff
@@ -391,7 +391,7 @@ ifeq ($(AUTOSPAWN), true)
 	@scripts/setup_integration_env.sh humio start
 	sleep 10 # Many services are very slow... Give them a sec..
 endif
-	${MAYBE_ENVIRONMENT_EXEC} cargo test --no-fail-fast --no-default-features --features humio-integration-tests --lib "::humio::.*::integration_tests::"
+	${MAYBE_ENVIRONMENT_EXEC} cargo test --no-fail-fast --no-default-features --features humio-integration-tests --lib ::humio::
```

Comment:
This runs all tests, no? Not just the integration tests.

Granted, that's not a big deal, but that is a difference from the other integration test make tasks.

Reply:
Ah, you beat me to it. Just added to the PR description:

> Technically, the new filter is too loose, but we currently do so in the other integration tests as well.

Reply:
Currently almost all of them are `--features $X-integration-tests --lib ::$X::`

Reply:
Aha, I am mistaken. I see that now. Thanks for pointing it out.